tests/e2e/normal_operations.go

```diff
@@ -20,8 +20,8 @@ func (k CCVTestSuite) TestHistoricalInfo() {
 	// and then call track historical info in the next block
 	createVal := func(k CCVTestSuite) {
 		// add new validator to consumer states
-		pk := ed25519.GenPrivKey().PubKey()
-		cVal, err := consumertypes.NewCCValidator(pk.Address(), int64(1), pk)
+		cId := crypto.NewCryptoIdentityFromRandSeed()
+		cVal, err := consumertypes.NewCCValidator(cId.SDKValAddress(), int64(1), cId.SDKPubKey())
```

Comment:
1st arg I'm not sure about. I think it might need to be `cId.ABCIAddressBytes()`
3rd arg is fine